src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala

```diff
@@ -56,6 +59,8 @@ class JdbcReadJournal(config: Config)(implicit val system: ExtendedActorSystem)
   val db = SlickDatabase.forConfig(config, readJournalConfig.slickConfiguration)
   sys.addShutdownHook(db.close())
 
+  val eventAdapters = Persistence(system).adaptersFor(JdbcAsyncWriteJournal.Identifier)
```

Comment:
I'm for the configuration solution. Please use the strategy used in the akka-persistence-inmemory plugin that uses a configuration called `write-plugin` as implemented in this commit: https://github.com/dnvriend/akka-persistence-inmemory/commit/04884163cdad89860039055766084365b91aa53e#diff-ecbc52c290666a9af5901127454d2bb8R59